README.md

```diff
@@ -17,6 +17,7 @@ npm install credit-card-type
 ```javascript
 var creditCardType = require("credit-card-type");
 
+// The card number provided should be normalized prior to usage here. 
 var visaCards = creditCardType("4111");
 console.log(visaCards[0].type); // 'visa'
```

Comment:
Should we have the console log here or was it just for debugging?

Reply:
This is in the readme, so it's just showing how to access the `type` parameter.

Reply:
This is in the docs, so it's fine being there.